mysite/static/udaan/assets/css/font-awesome.css

```diff
@@ -0,0 +1,303 @@
+/*  Font Awesome
```

Comment:
@PiyushPI07 any particular reason for not using a CDN instead?

Reply:
Not really, font-awesome isnt really being used. I'll remove the import.